PactNet.Tests/PactNet.Tests.csproj

```diff
@@ -1,6 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net46</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' " >
+      netcoreapp2.0

```

Comment:
Any reason to target `netcoreapp2.0` and not `netcoreapp1.1` like we were originally doing?

Reply:
It did occur to me that .NET core 2.0 is much more heavily used than 1.1, and so running the tests targeting that would likely be more representative of real world usage - if you're not happy to up-version then I'll change it back to 1.1 - sorry, should really have mentioned this explicitly on the PR.

Reply:
My thought behind this was have the tests target the lowest common denominator. My assumption here is if it passes in 1.1 it should also pass in 2.0.